README.md

```diff
@@ -1 +1,50 @@
 # codacy-clang-tidy
+
+A standalone tool that converts [Clang-Tidy](https://clang.llvm.org/extra/clang-tidy/)
+reports to Codacy's format. It allows the integration of Clang-Tidy into your Codacy workflow.
+
+## Usage
+
+TODO
```

Comment:
Prease

Reply:
There is a separate task for this. It'll be done in a PR that should be open very soon

Reply:
As I commented on the PR description, I'll to this in another PR